packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -205,25 +213,40 @@ export class IndexedDbPersistence implements Persistence {
 
   private queryCache: IndexedDbQueryCache;
   private remoteDocumentCache: IndexedDbRemoteDocumentCache;
+  private webStorage?: Storage;
+  private listenSequence: ListenSequence;
 
   constructor(
     private readonly persistenceKey: string,
     private readonly clientId: ClientId,
     platform: Platform,
     private readonly queue: AsyncQueue,
     serializer: JsonProtoSerializer,
-    synchronizeTabs: boolean
+    private readonly multiClientParams?: {
+      sequenceNumberSyncer: SequenceNumberSyncer;
+    }
   ) {
     this.dbName = persistenceKey + IndexedDbPersistence.MAIN_DATABASE;
     this.serializer = new LocalSerializer(serializer);
     this.document = platform.document;
     this.window = platform.window!;
-    this.allowTabSynchronization = synchronizeTabs;
+    if (multiClientParams !== undefined) {
```

Comment:
Hm. Not sure I am big fan since this used to be very explicit and is now very implicit.

Reply:
As discussed, this will eventually become mandatory, and the flag will go away. If we can come up with a way to not allow invalid configurations while also being more explicit, I'm happy to implement it. But, I would like to avoid being able to specify invalid configurations.

On a related note, I would like to eventually get rid of external callers calling the constructor followed immediately by `start()`. I think that would be a good time to switch to static methods that return a `Promise<IndexedDbPersistence>`, and it would be reasonable to name one of them such that it's obviously creating a multi-client instance.